src/vanilla/utils/atomWithStorage.ts

```diff
@@ -161,15 +161,23 @@ export function createJSONStorage<Value>(
         callback(newValue)
       })
 
-  let subscriber = getStringStorage()?.subscribe
+  let subscriber: StringSubscribe | undefined
+  try {
+    subscriber = getStringStorage()?.subscribe
+    // getStringStorage() //?.getItem('')
```

Comment:
```suggestion
```

Reply:
Ah, this wasn't meant to be in the commit.